.github/workflows/build.yaml

```diff
@@ -17,13 +18,17 @@ jobs:
 
       - name: Set buildx alias
         run: docker buildx install
+        
+      - name: Install docker pushrm
```

Comment:
nice one :)

Reply:
It's not working correctly though, the READMEs are not pushed by the pipeline. I'll investigate.

Reply:
Fixed.